package.json

```diff
@@ -26,6 +26,7 @@
   "devDependencies": {
     "@changesets/changelog-github": "^0.4.8",
     "@changesets/cli": "^2.26.2",
+    "@farcaster/hub-nodejs": "^0.11.0",
```

Comment:
Is this a peer dependency?

Also this library is quite massive https://bundlephobia.com/package/@farcaster/hub-nodejs@0.11.0

As OnchainKit is used internally at Coinbase, we have strict requirements to support both web, react native and node. And keep the dependency and kbs of each feature extremely lean.

Can we operarte the sam result but without this library? Or maybe with a different one?

Reply:
This was a mistake. I meant to remove it. Will do that in the next push.